Classes/Domain/Dto/TranslateOptions.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WebVision\WvDeepltranslate\Domain\Dto;
+
+final class TranslateOptions
```

Comment:
Should we mark this as `@internal` for now - or is this considered public extension API ?